app/src/main/java/com/example/munchkin/Karte/SchatzkartenStapel.java

```diff
@@ -35,9 +35,10 @@ public Karte getKarte()
         return Schatzkarte.getRandomSchatzkarte();
     }
 
-
     void onSchatzkartenStapelClicked()
     {
-        Player.getLocalPlayer().getInventar().getHandKarten().addKarte(getKarte());
+        Karte gehobeneKarte = getKarte();
+        gehobeneKarte.onKarteGehoben();
```

Comment:
onKarteGehoben ist nicht notwendig, da beim heben von KArten nie ein Event getriggert wird, sondern diese direkt in die Handkarten aufgenommen werden